presto-hive/src/main/java/com/facebook/presto/hive/HiveWriteUtils.java

```diff
@@ -286,7 +286,7 @@ else if (isMapType(type)) {
         else if (isRowType(type)) {
             return ObjectInspectorFactory.getStandardStructObjectInspector(
                     type.getTypeSignature().getParameters().stream()
-                            .map(parameter -> parameter.getNamedTypeSignature().getName())
+                            .map(parameter -> parameter.getNamedTypeSignature().getName().get())
```

Comment:
`orThrow()` is safer. Especially when you collect `toList()`

Reply:
How so?

Reply:
Sorry about that. Somehow I thought that `Optional.empty()` would return `null` from `get()` method.